src/TesApi.Web/Startup.cs

```diff
@@ -176,6 +185,18 @@ public void Configure(IApplicationBuilder app)
                         logger.LogInformation("Configuring for Production environment");
                         return r;
                     });
+
+        private static bool IsAvailableAzureEnvironmentName(string azureName)
```

Comment:
please make the same changes as specified in AzureEnvironmentExtension.cs

Reply:
Fixed
```csharp
        public static bool IsEnvironmentNameAvailable(string envName)
        {
            foreach (var azEnv in AzureEnvironment.KnownEnvironments)
            {
                if (string.Equals(azEnv.Name, envName, StringComparison.OrdinalIgnoreCase))
                {
                    return true;
                }
            }
            return false;
        }
```